src/main/java/org/broadinstitute/dsm/route/DisplaySettingsRoute.java

```diff
@@ -1,5 +1,7 @@
 package org.broadinstitute.dsm.route;
 
+import com.google.gson.Gson;
+import com.sun.xml.bind.v2.runtime.output.SAXOutput;
```

Comment:
I think this was just automatically imported by IDE and is not used in this class at all.